src/tcgwars/logic/util/CardList.groovy

```diff
@@ -61,6 +61,11 @@ public class CardList extends ArrayList<Card> {
   public CardList copy(){
     return new CardList(this);
   }
+  public CardList shuffledCopy(){
```

Comment:
This is an extra shuffledCopy(), there is another function already defined in line 70

Reply:
Welp, guess that's one thing to lookout when rebase-merging these PRs (tested a card in master with the method)